docs/source/guides/export_model.mdx

```diff
@@ -168,42 +170,35 @@ optimum-cli export neuron --model local_path --task question-answering --batch_s
 
 Note that providing the `--task` argument for a model on the Hub will disable the automatic task detection. The resulting `model.neuron` file, can then be loaded and run on Neuron devices. 
 
-## Exporting a model to Neuron using the export function
+## Exporting a model to Neuron using `NeuronModelForXXX` APIs
 
-You will also be able to export your models to Neuron format directly with the export function, and load it for inference. Here is an example:
+You will also be able to export your models to Neuron format with `optimum.neuron.NeuronModelForXXX` model classes. Here is an example:
 
 ```python
-import torch
-from pathlib import Path
-from transformers import AutoTokenizer, AutoModelForSequenceClassification
-from optimum.exporters.neuron import export
-from optimum.exporters.neuron.model_configs import DistilBertNeuronConfig
-
-model_id = "distilbert-base-uncased-finetuned-sst-2-english"
-sequence = "Hello, my cats are cute."
-tokenizer = AutoTokenizer.from_pretrained(model_id)
-model = AutoModelForSequenceClassification.from_pretrained(model_id)
-neuron_config = DistilBertNeuronConfig(
-    config=model.config, task="text-classification", batch_size=1, sequence_length=18, dynamic_batch_size=True
-)
-output_path = Path("test_bert/model.neuron")
-
-# Export to Neuron model
-export(
-    model=model,
-    config=neuron_config,
-    output=output_path,
-    auto_cast="all",
-    auto_cast_type="bf16",
-)
-
-# Reload it for inference
-neuron_model = torch.jit.load(output_path)
-example = tokenizer.encode_plus(
-    sequence, max_length=neuron_config.sequence_length, padding="max_length", return_tensors="pt"
-)
-
-logits = neuron_model(*tuple(example.values()))
+>>> from optimum.neuron import NeuronModelForSequenceClassification
+
+>>> input_shapes = {"batch_size": 1, "sequence_length": 64}  # mandatory shapes
+>>> model = NeuronModelForSequenceClassification.from_pretrained(
+...   "distilbert-base-uncased-finetuned-sst-2-english", export=True, **input_shapes
+... ) 
+
+# Save the model
+>>> model.save_pretrained("./distilbert-base-uncased-finetuned-sst-2-english_neuron/")
```

Comment:
does that save the config?

Reply:
Yes it does: https://github.com/huggingface/optimum/blob/main/optimum/modeling_base.py#L124